uclapi.openapi.json

```diff
@@ -4244,6 +4810,1679 @@
+                      "type": "string",
+                      "description": "Will be returned if there is an error retrieving the given LibCal space, e.g. if the space ID provided is invalid",
+                      "example": "Zone not found."
+                    },
+                    "zone": {
+                      "allOf": [
+                        { "$ref": "#/components/schemas/libcal_zone" }
+                      ],
+                      "properties": {
+                        "itemIds": {
+                          "type": "array",
+                          "items": {
+                            "$ref": "#/components/schemas/libcal_space_id"
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "400" : {
+            "description" : "Request error",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "$ref" : "#/components/schemas/error"
+                },
+                "examples" : {
+                  "1" : {
+                    "$ref" : "#/components/examples/ErrorNoToken"
+                  },
+                  "2" : {
+                    "$ref" : "#/components/examples/ErrorTokenNotExist"
+                  },
+                  "3" : {
+                    "$ref" : "#/components/examples/ErrorInvalidToken"
+                  },
+                  "4" : {
+                    "$ref" : "#/components/examples/ErrorThrottling"
+                  },
+                  "5" : {
+                    "$ref" : "#/components/examples/ErrorNoClientSecret"
+                  },
+                  "6" : {
+                    "$ref" : "#/components/examples/ErrorInvalidClientSecret"
+                  },
+                  "7" : {
+                    "$ref" : "#/components/examples/ErrorInactiveToken"
+                  },
+                  "8" : {
+                    "$ref" : "#/components/examples/ErrorPersonalData"
+                  },
+                  "9" : {
+                    "$ref" : "#/components/examples/ErrorRejectedScope"
+                  },
+                  "10": {
+                    "$ref": "#/components/examples/ErrorLibCalOAuthTokenInvalid"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/libcal/space/bookings": {
+      "get": {
+        "summary": "Get all LibCal bookings",
+        "tags": [ "LibCal" ],
+        "parameters": [
+          {
+            "name": "eid",
+            "in": "query",
+            "description": "Comma delimited list of LibCal space IDs. If specified, will only show bookings for those spaces.",
+            "required": false,
+            "example": "469,470",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "seat_id",
```

Comment:
I think we use this multiple times so it might be better to move it into a `schema` and reference it whenever we need it?

Although I think at this point, this file is getting ridiculously big so this might be something to do after splitting the spec into multiple files -- see #3504